alpakaConfig.cmake

```diff
@@ -315,6 +315,12 @@ IF(ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE OR ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE OR A
         SET(ALPAKA_ACC_CPU_BT_OMP4_ENABLE OFF CACHE BOOL "Enable the OpenMP 4.0 CPU block and thread back-end" FORCE)
 
     ELSE()
+
+	    # Check whether only OpenMP 2 is supported
+	    IF(OpenMP_CXX_VERSION_MAJOR STREQUAL "2")
```

Comment:
Could you please try if something like `IF(${OpenMP_CXX_VERSION_MAJOR}.${OpenMP_CXX_VERSION_MINOR} VERSION_LESS 4.0)` works? Mabe this needs some " other other tricks.